src/client/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest AS build
```

Comment:
We should specify the direct app supported (run ng version in CLI) and LTS node version (current long time support version can be checked here https://github.com/nodejs/Release)